### PR TITLE
GA Deployment Description updates

### PR DESCRIPTION
**Pull request by dvargas92495**

### CODE REVIEW

File: fern/docs/content/change-log/2024-08.mdx [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 ---
 
+## Deployment Descriptions
+
+_August 2nd, 2024_
+
+You can now update your Prompt and Workflow Deployments to include a human-readable description. This is useful for giving other members of your team a high level summary
+of what the Prompt or Workflow does without needing to parse through the configuration or control flow.
+
+![Evaluation Report History](https://storage.cloud.google.com/vellum-public/help-docs/changelogs/2024-08/update-deployment-description.png)
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:12]: fixing alias

**Comment by noanflaherty** [comment]
Can we include a screenshot if where this description appears once set?

**Comment by dvargas92495** [comment]
rerequesting review @noanflaherty

File: fern/docs/content/change-log/2024-08.mdx [hunk 2]
```diff
@@ -2,6 +2,19 @@
 title: Changelog | August, 2024
 ---
 
+## Deployment Descriptions
+
+_August 2nd, 2024_
+
+You can now update your Prompt and Workflow Deployments to include a human-readable description. This is useful for giving other members of your team a high level summary
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-08.mdx:9]:
```suggestion
You can now update your Prompt and Workflow Deployments to include a human-readable description. This is useful for giving other members of your team a high-level summary
```

**Pull request merged**